examples/complete-integration/app/server.ts

```diff
@@ -0,0 +1,74 @@
+import express from "express";
+import bodyParser from "body-parser";
+import {
+  configureTestBox,
+  TestBoxUseCaseRequest,
+  TestBoxTrial,
+  TestBoxTrialRequest,
+} from "@testboxlab/node-sdk";
+
+configureTestBox({
+  productId: "my-slugified-product-name",
+});
+
+const app = express();
+app.use(bodyParser.json())
+
+app.post("/api/testbox/trial", async (req, res) => {
+  const trialRequest = await TestBoxTrialRequest.fromExpressRequest(req);
+
+  const authorization = req.headers["authorization"];
+  if (!authorization) return res.status(401).send();
+
+  const tokenVerified = await trialRequest.verifyToken(authorization);
+  if (!tokenVerified) {
+    // The token verification failed, meaning someone is trying to pretend to be
+    // TestBox! Do not process their request.
+    return res.status(401).send();
+  }
+
+  // Once you have provisioned an account, we need to start telling TestBox about it.
+  const testboxTrial = new TestBoxTrial();
```

Comment:
I think we should probably call this TestboxTrial instead of TestBoxTrial. We often treat testbox as one word (like you did with the variable)

Reply:
Actually, ignore this. It looks like we do separate words everywhere so this is fine